docs/releases/1.9.txt

```diff
@@ -303,3 +303,11 @@ removed in Django 1.9 (please see the :ref:`deprecation timeline
 * Private API ``django.test.utils.TestTemplateLoader`` is removed.
 
 * The ``django.contrib.contenttypes.generic`` module is removed.
+
+* ``django.http.responses.REASON_PHRASES`` and
```

Comment:
This documentation is in the wrong section -- these are features that have gone through a deprecation cycle. The section you want is "Backwards incompatible changes"